slack/web/classes/blocks.py

```diff
@@ -379,3 +381,31 @@ def __init__(
         self.call_id = call_id
         self.api_decoration_available = api_decoration_available
         self.call = call
+
+
+class HeaderBlock(Block):
+    type = "header"
+    text_max_length = 3000
+
+    @property
+    def attributes(self) -> Set[str]:
+        return super().attributes.union({"text"})
+
+    def __init__(
+        self,
+        *,
+        block_id: Optional[str] = None,
+        text: Union[str, dict, TextObject] = None,
+        **others: dict,
+    ):
+        """A header is a plain-text block that displays in a larger, bold font.
+        https://api.slack.com/reference/block-kit/blocks#header
+        """
+        super().__init__(type=self.type, block_id=block_id)
+        show_unknown_key_warning(self, others)
+
+        self.text = TextObject.parse(text)
+
+    @JsonValidator("text or fields attribute must be specified")
+    def _validate_text_or_fields_populated(self):
+        return self.text is not None or self.fields
```

Comment:
This block doesn't have fields.
```suggestion
    def _validate_text__populated(self):
        return self.text is not None
```

Reply:
Keen eye @seratch! I borrowed the implementation from another block and didn't notice that it also validated against fields.

Thanks for the change suggestion! Applied in commit 1296be437375ac1f908df9059cfb4e6ef038b4df and tests pass.